components/contribution-flow/ContributionFlowHeader.js

```diff
@@ -33,9 +35,22 @@ class NewContributionFlowHeader extends React.Component {
   };
 
   render() {
-    const { collective, isEmbed } = this.props;
+    const { collective, isEmbed, LoggedInUser } = this.props;
     const contributors = collective.contributors?.nodes;
 
+    let loggedInUserIsContributor = false;
+    let loggedInUserTotalDonations = 0;
+
+    if (contributors) {
+      loggedInUserIsContributor = contributors.find(
```

Comment:
We can't rely on `contributors`. If you look at the GraphQL query, you'll notice that we're fetching only 6 of them: https://github.com/opencollective/opencollective-frontend/blob/main/components/contribution-flow/graphql/fragments.js#L54. If your account is not included in these 6 contributors, you won't see the number displayed.

We don't want to fetch more contributors just for this (that would be extremely heavy - a collective like webpack probably has more than 10.000 contributors) so we'll need a clever approach that fetches just your existing memberships to this collective. I can recommend adding a new query to the composition in `pages/contribution-flow.js` like:

```gql
{
  loggedInAccount {
    memberOf(account: {slug: "webpack"}) {
      nodes {
        id
        stats {
          totalAmountContributed
        }
      }
    }
  }
}
```

(`member.stats` doesn't exist yet and would need to be added).

Reply:
`totalDonations` was already available.

```js
loggedInAccount {
      memberOf(account: { slug: "webpack" }) {
        nodes {
          id
          totalDonations {
            value
          }
        }
      }
    }
 ```